### DISCUSSION TITLE
Accept table for bullets for each level.

### DISCUSSION THREAD

**Discussion by allreddv** [MeanderingProgrammer/render-markdown.nvim#217] [Ideas]
First, thanks for the amazing plugin, I use it everyday for note taking.  

This is just a feature request if possible.  

Instead of only being able to enter in a single value for each level of bullets, if we could add a tables for each level instead so we could do a traditional outline format. 

So like for first bullet we would have the option to enter table with Roman Numerals, then second lvl ABC, etc. And for each line it advances through the table until you go to next lvl where it starts advancing again.  

I have tried using bullets.vim in combination but can't for the life of me get treesitter to see them as anything other than a paragraph so I can't get highlighting as list items  and i would like to have the bullets colored to stand out. 

Idealy i would like, 

Roman Numeral uppercase
ABC uppercase
Numbers
abc lowercase
Roman numeral lowercase
- - - 
* * *
+ + + 

I have tried to find a way to get this to work on my own but have not had any success.  Skill issues.  

Anyways, just a wish item.  Thanks

**Comment by MeanderingProgrammer** [comment]
Sure, added this here: https://github.com/MeanderingProgrammer/render-markdown.nvim/commit/04e75a35900d9e6bc53eb0f18e2aeee140d6c5ae

**Comment by allreddv** [comment]
Wow,  Amazing, Thank you so much.
